packages/errors/api-report.md

```diff
@@ -14,6 +14,17 @@ export class AuthenticationError extends CustomErrorBase {}
 // @public
 export class ConflictError extends CustomErrorBase {}
 
+// @public
+export type ConsumedResponse = {
```

Comment:
Possibly general enough to have in `@backstage/types`

Reply:
Not now right? :p

Reply:
Just one more breaking change down the line :)

Reply:
Inlining does have benefits but it also makes the intent less clear; the name at least conveys something useful. If we could have marked it as alpha or internal or something that might have helped. I've approved, I think it's fine as is, just airing some loose thoughts